### DISCUSSION TITLE
Feature: Product Rating System.

### DISCUSSION THREAD

**Discussion by CovasJJ** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#5] [Ideas]
I believe a system of leaving reviews on each product is also very important for any online market since it provides feedback for the sellers. These reviews may also help convince the buyers to make their purchase. A section in which users can see the highest-rated products could also be added to the feed of all the products.

**Comment by david-baez-bravo** [comment]
There should be tags for products part of a ranking leaderboard. Like number 2 highest rated notebook seller or number 1 in jewelry. But not only by rating but also top sellers.

**Reply by Kaariinaa08** [reply]
Building on the idea of a Product Rating system, I believe we should also consider implementing a Buyer Rating system. This would be particularly valuable for transactions that require in-person pickups, where trust between the buyer and seller is crucial.
